queries.sql

```diff
@@ -1,3 +1,23 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+
+SELECT * from animals WHERE name LIKE '%mon'; 
+
+SELECT name from animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth < '2019-01-01';
+
+SELECT name from animals WHERE neutered = true and escape_attempts < 3;
+
+SELECT date_of_birth from animals WHERE name = 'Agumon' or  name = 'Pikachu';
+
+SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
+
+SELECT * from animals WHERE neutered = true;
+
+SELECT * from animals WHERE name <> 'Gabumon';
```

Comment:
- [ ] Great effort working on the change requested by the previous reviewer, I would kindly suggest you had the correct screenshot of the `queries` result in your pull request. The one you have there is for the one before implementing the changes, kindly replace it with the new result. 👍 👍